CHANGELOG.MD

```diff
@@ -1,3 +1,8 @@
+## May 6, 2024
+
+- **Feature** Auto-populate question descriptions [🎟️ DESENG-596](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-596)
```

Comment:
From the Changelog, and by merging into Main, it looks like this feature is complete just by upgrading the version. Is that the case?

Reply:
Yes the changes were pushed to met-formio and published to npm. This version of met-formio has the changes needed for this ticket. I update the changelog to have more details.

Reply:
Ohhh okay so the substantive changes were in the met-formio repository for this. Thanks for clarifying